diff --git a/src/Controller/ReportController.php b/src/Controller/ReportController.php
index 9ff362a..d39812f 100755
--- a/src/Controller/ReportController.php
+++ b/src/Controller/ReportController.php
@@ -64,6 +64,8 @@ class ReportController extends ControllerBase {
    */
   public function overview() {
 
+    include_once DRUPAL_ROOT . '/core/includes/install.inc';
+
     $build['report'] = [
       '#type' => 'status_report',
       '#requirements' => [],
@@ -83,8 +85,6 @@ class ReportController extends ControllerBase {
       return $build;
     }
 
-    include_once DRUPAL_ROOT . '/core/includes/install.inc';
-
     $start = microtime(TRUE);
 
     $info = $this->redis->info();
