diff --git a/modules/webform_access/src/Breadcrumb/WebformAccessBreadcrumbBuilder.php b/modules/webform_access/src/Breadcrumb/WebformAccessBreadcrumbBuilder.php
index ec7a068f2..8af228a3e 100644
--- a/modules/webform_access/src/Breadcrumb/WebformAccessBreadcrumbBuilder.php
+++ b/modules/webform_access/src/Breadcrumb/WebformAccessBreadcrumbBuilder.php
@@ -43,6 +43,11 @@ class WebformAccessBreadcrumbBuilder implements BreadcrumbBuilderInterface {
     if (strpos($route_name, 'webform_access') === FALSE) {
       return FALSE;
     }
+
+    if (!$route_match->getRouteObject()) {
+      return FALSE;
+    }
+
     $path = Url::fromRouteMatch($route_match)->toString();
 
     if (strpos($path, 'admin/structure/webform/access/') === FALSE) {
